File: Backend/DTOs/PizzaPedidoDto.cs
using JetBrains.Annotations;

namespace Pizzaria.DTOs;

[PublicAPI]
public struct PizzaPedidoDto
{
    public List<string> Sabores { get; set; } = new();
    public string Tamanho { get; set; }
    public int Quantidade { get; set; }

    public PizzaPedidoDto(List<string> sabores, string tamanho, int quantidade)
    {
        Sabores = sabores;
        Tamanho = tamanho;
        Quantidade = quantidade;
    }
}

File: Frontend/src/app/models/dtos/PizzaPedidoDto.ts
export class PizzaPedidoDto {
  sabores : string[];
  tamanho: string;
  quantidade: number;

  constructor(sabores: string[], tamanho: string, quantidade: number = 1) {
    this.sabores = sabores;
    this.tamanho = tamanho;
    this.quantidade = quantidade;
  }
}
